build/charts/antrea/templates/agent/daemonset.yaml

```diff
@@ -260,6 +260,8 @@ spec:
             mountPropagation: HostToContainer
           - name: xtables-lock
             mountPath: /run/xtables.lock
+          - name: host-pod-resources
+            mountPath: /var/lib/kubelet/pod-resources
```

Comment:
I see that Multus mounts all of `/var/lib/kubelet`, but they probably need more things than we do:

https://github.com/k8snetworkplumbingwg/multus-cni/blob/fba1fea81e0d4040244badf9d8c8e0eaddc717e8/deployments/multus-daemonset-thick.yml#L181-L183

Would you see any reason to also set `mountPropagation` here?